src/stores/business.ts

```diff
@@ -313,9 +313,7 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
       }
 
       case AllowableActionE.BUSINESS_SUMMARY: {
-        // NB: specific entities are targeted via LaunchDarkly
-        const ff = !!getFeatureFlag('supported-business-summary-entities')?.includes(legalType)
-        return (ff && isBusiness)
+        return (isBusiness)
```

Comment:
Although this code (and the code below in dashboardActions.ts) is never used for this action, it's good that you're leaving it in, because there a feature coming soon to make Business Summary conditional on a business flag.

Of course, Links.vue will need to be updated to call `isAllowed()`.

FYI.